cdap-docs/admin-manual/source/installation/azure-hdinsight.rst

```diff
@@ -0,0 +1,241 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :copyright: Copyright © 2017 Cask Data, Inc.
+
+.. :section-numbering: true
+
+.. _admin-installation-azure-hdinsight:
+
+=========================================
+Installation on Microsoft Azure HDInsight
+=========================================
+
+Introduction
+============
+
+This section describes installing CDAP on `Microsoft Azure HDInsight
+<https://azure.microsoft.com/en-us/services/hdinsight/>`__ using
+Microsoft's website to:
+
+- :ref:`Create an HDInsight HBase cluster <admin-installation-azure-hdinsight-creating-cluster>` with CDAP installed as an application
+- Install CDAP as an application :ref:`on an existing HDInsight HBase cluster <admin-installation-azure-hdinsight-installing-on-existing>`
+- :ref:`Verify that CDAP is installed and running <admin-installation-azure-hdinsight-verification>` by using the CDAP UI
+
+Information on Microsoft Azure HDInsight is `available online
+<https://docs.microsoft.com/en-us/azure/hdinsight/>`__.
+
+CDAP |short-version| is compatible with Microsoft Azure HDInsight |hdinsight-versions|.
+
+
+.. _admin-installation-azure-hdinsight-creating-cluster:
+
+Creating a Cluster with CDAP Installed
+======================================
+
+1. Log onto Microsoft Azure and navigate to *HDInsight clusters* (either through the *New*
+   menu and under *Intelligence + analytics*, or directly through the *HDInsight clusters*
+   icon in the left sidebar):
+
+   .. figure:: ../_images/azure-hdinsight/azure-hdinsight-1.png
+      :figwidth: 100%
+      :width: 611px
+      :class: bordered-image
+
+      **Microsoft Azure:** Finding HDInsight
+
+#. Create an *HDInsight cluster* by clicking the "Create" button. *Quick create* will create a
+   cluster, but without CDAP installed. We recommend instead using the *Custom (size,
+   settings, apps)* tab to access additional options and add installing CDAP as part of
+   the cluster creation.
+
+   .. figure:: ../_images/azure-hdinsight/azure-hdinsight-2.0.png
+      :figwidth: 100%
+      :width: 800px
+      :class: bordered-image
+
+      **Microsoft Azure HDInsight:** Creating a cluster using *Quick create*
+
+   Using the *Custom (size, settings, apps)* tab, enter a *Cluster name* (such as
+   *cdap-cluster*). Under *Cluster type*, select "HBase". (To continue, click *Select* to
+   close the *Cluster configuration* window):
+
+   .. figure:: ../_images/azure-hdinsight/azure-hdinsight-2.1.png
+      :figwidth: 100%
+      :width: 800px
+      :class: bordered-image
+
+      **Microsoft Azure HDInsight:** Creating a cluster using *Custom (size, settings, apps)* settings
+
+#. Enter the required basic settings: *Cluster login password*, *Resource group*, and *Location*.
+   As a minimum, a cluster requires 4 D3 v2 region nodes. This is enough YARN capacity to
```

Comment:
This size requirement is a duplicate of step 6 below, where it belongs.  not needed here.